code/modules/power/cell.dm

```diff
@@ -10,6 +10,8 @@
 	throw_speed = 2
 	throw_range = 5
 	w_class = 2
+	var/part_type = "cell" //I blame vehicles
```

Comment:
I do, creating variables that aren't really being used as variables is not good practice, and incredibly dirty. You could do an istype check and add a bit of extra (admittedly snowflakey) logic rather than doing this.